pkg/kv/bulk/sst_batcher.go

```diff
@@ -310,6 +317,11 @@ func (b *SSTBatcher) flushIfNeeded(ctx context.Context, nextKey roachpb.Key) err
 
 // Flush sends the current batch, if any.
 func (b *SSTBatcher) Flush(ctx context.Context) error {
+	if err := b.grp.Wait(); err != nil {
```

Comment:
quick comment that we're waiting for addsstable requests across ranges to return

Reply:
think the name helps here?